### PR TITLE
chore: add pr and issue templates, codeowners and contributing.md

### PR DESCRIPTION
**Pull request by JackHamer09**
Added:
 - Bug report and feature request issue templates
 - Codeowners and contributing guidelines
 - Pull request template

### CODE REVIEW

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -0,0 +1,10 @@
+# This CODEOWNERS file sets the individuals responsible for code in the zksync-hardhat-template repository.
+
+# These users are the default owners for everything in the repo.
+# They will be requested for review when someone opens a pull request.
+* @matter-labs/devxp
```
Comment (MexicanAce) [inline .github/CODEOWNERS:5]: I'm hesitant to add the `.github/` directory, as this is a template repo that others are expected to copy/paste its contents.

At the very least, we shouldn't merge this PR until we update `zksync-cli` accordingly: https://github.com/matter-labs/zksync-cli/blob/main/src/commands/create/create.ts#L76-L80

Comment (JackHamer09) [reply .github/CODEOWNERS:5]: Ah, right, will work on that

Comment (JackHamer09) [reply .github/CODEOWNERS:5]: Should be fixed in https://github.com/matter-labs/zksync-cli/pull/73

Comment (JackHamer09) [reply .github/CODEOWNERS:5]: Fixed!

**Review by MexicanAce — APPROVED** [review]
LGTM

**Pull request merged**